scripts/install_third_party_libs.py

```diff
@@ -24,17 +24,23 @@
 
 # These libraries need to be installed before running or importing any script.
 TOOLS_DIR = os.path.join(os.pardir, 'oppia_tools')
-# Download and install pyyaml.
-if not os.path.exists(os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'pyyaml==5.1.2', '--target',
-        os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')])
-
-# Download and install future.
-if not os.path.exists(os.path.join('third_party', 'future-0.17.1')):
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', 'future==0.17.1', '--target',
-        os.path.join('third_party', 'future-0.17.1')])
+
+
+PREREQUISITES = [
+    ('pyyaml', '5.1.2', os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')),
+    ('future', '0.17.1', os.path.join('third_party', 'future-0.17.1')),
+]
+
+for PACKAGE, VERSION, PATH in PREREQUISITES:
```

Comment:
the `PACKAGE`, `VERSION`, and `PATH` don't need to be in all caps.

Reply:
Also, put this into a function.

Reply:
Alright! Also I was not able to figure out how to get 100% test coverage. I tried raising exception in `mock_check_call` after conditions like` if str(self.commands[-1]) == '--system':` but I was not able to. Can you help me do that. do I need to use pipe for it?

Reply:
Done! put it into a function and removed all caps.

Reply:
`PACKAGE`, `VERSION`, and `PATH` are still in all caps.

Reply:
> Alright! Also I was not able to figure out how to get 100% test coverage. I tried raising exception in `mock_check_call` after conditions like` if str(self.commands[-1]) == '--system':` but I was not able to. Can you help me do that. do I need to use pipe for it?

Sorry, I missed this comment. Can you elaborate more on that question?

Reply:
I was taking another approach before where I was trying to use only subprocess.check_call. That gave me errors so I switched to popen where I could check what the error message is.